Changelog.md

```diff
@@ -1,5 +1,9 @@
 ## 5.0.3 - Unrelesed
 
+### Added
+
+-   JLink Warning when Application does not find the expected JLink version.
+
 ### Updated
 
 -   Bundeled J-Link Version to 7.94i for windows.
```

Comment:
```suggestion
-   Bundled SEGGER J-Link to v7.94i for Windows.
```

Reply:
FYI: I mostly used that suggestion in #1044 now.